renderer/index.js

```diff
@@ -533,11 +534,25 @@ function closePlayer () {
   if (state.window.isFullScreen) {
     dispatch('toggleFullScreen', false)
   }
+
+  unblockPowerSave()
   restoreBounds()
   stopServer()
   update()
 }
 
+function blockPowerSave () {
+  powerSaveBlockIDs.push(powerSaveBlocker.start('prevent-display-sleep'))
```

Comment:
why not have just one `powerSaveBlockID`?

Then, in `blockPowerSave`:

```js
if (!powerSaveBlocker.isStarted(powerSaveBlockID)) {
  powerSaveBlockID = powerSaveBlocker.start(...)
}
```

...and similarly in `unblockPowerSave`:

```js
if (!powerSaveBlocker.isStarted(powerSaveBlockID)) {
  powerSaveBlocker.stop(powerSaveBlockID)
}
```